runtime/vm.go

```diff
@@ -164,24 +156,20 @@ func (v *VM) Run() (err error) {
 	return nil
 }
 
-func (v *VM) run() (verr error) {
+func (v *VM) run() error {
 	defer func() {
 		if r := recover(); r != nil {
-			v.valid = false
+			v.lastPanic = r
```

Comment:
Set v.aborting to 1. I think in the last case the vm will continue past the OpCall if Go swallows the panic and doesn't return an error (even though v.err is not empty)